src/gui/processing/qgsprocessingdxflayerswidgetwrapper.cpp

```diff
@@ -0,0 +1,332 @@
+/***************************************************************************
+  qgsprocessingdxflayerswidgetwrapper.cpp
+  ---------------------
+  Date                 : September 2020
+  Copyright            : (C) 2020 by Alexander Bruy
+  Email                : alexander dot bruy at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsprocessingdxflayerswidgetwrapper.h"
+
+#include <QBoxLayout>
+#include <QLineEdit>
+#include <QMessageBox>
+#include <QPushButton>
+#include <QStandardItemModel>
+#include <QToolButton>
+
+#include "qgspanelwidget.h"
+#include "qgsprocessingcontext.h"
+#include "qgsprocessingparameters.h"
+#include "qgsprocessingoutputs.h"
+#include "qgsprocessingparameterdxflayers.h"
+
+/// @cond private
+
+//
+// QgsProcessingDxfLayerDetailsWidget
+//
+
+QgsProcessingDxfLayerDetailsWidget::QgsProcessingDxfLayerDetailsWidget( const QVariant &value, QgsProject *project )
+{
+  setupUi( this );
+
+  mFieldsComboBox->setAllowEmptyFieldName( true );
+
+  QgsProcessingContext context;
```

Comment:
this will need to be a member variable -- otherwise layers temporarily stored in the context by the call to variantMapAsLayer below will immediately be deleted at the end of the constructor, and mLayer will be a dangling pointer.

Reply:
Should be fixed. If I'm not wrong same issue then exists in vector tile parameter, I will submit another PR later to fix it too.